docs/runtimes/create-and-use-functions.ipynb

```diff
@@ -445,6 +446,24 @@
    "id": "b6467c47",
    "metadata": {},
    "source": [
+    "## Set requirements when creating a function\n",
+    "\n",
+    "You can list the Python dependencies, for example:\n",
+    "```python\n",
+    "# By providing a list of packages\n",
```

Comment:
This section can be moved to be before `Import or use an existing function`, so that all function creation related documentation is grouped together